src/lib/locale/locales.js

```diff
@@ -136,6 +136,12 @@ export function updateLocale(name, config) {
         if (locales[name] != null) {
             parentConfig = locales[name]._config;
         }
+        else {
```

Comment:
How about:

```js
tmpLocale = locales[name] || loadLocale(name);
if (locale != null) {
  parentConfig = tmpLocale._config;
}
```

Reply:
Looks good, I've committed with the suggested refactor.